src/pocketclaw/agents/errors.py

```
@@ -0,0 +1,258 @@
+        if isinstance(error, RateLimitError):
+            return True
+    except ImportError:
+        pass
+    return (
+        "rate_limit" in error_str
+        or "rate limit" in error_str
+        or "too many requests" in error_str
+    )
+
+
+def _is_ollama_error(error: Exception, error_str: str, error_type: str) -> bool:
```

Ruff (0.15.0)

[warning] Unused function argument: `error`

(ARG001)